examples/shared/base-ui/src/components/ui/context-menu.tsx

```
@@ -1,6 +1,6 @@
 'use client';
 
-import { cn } from '@examples/design-system/lib/utils';
+import { cn } from '@examples/base-ui/lib/utils';
```

ESLint

[error] Unable to resolve path to module '@examples/base-ui/lib/utils'.

(import-x/no-unresolved)